src/res_parsetree_viewer.ml

```diff
@@ -168,8 +192,9 @@ let filterParsingAttrs attrs =
       match attr with
       | ( {
             Location.txt =
-              ( "ns.ternary" | "ns.braces" | "res.template" | "bs" | "ns.iflet"
-              | "ns.namedArgLoc" | "ns.optional" );
+              ( "ns.ternary" | "ns.braces" | "res.template" | "res.await"
+              | "res.async" | "bs" | "ns.iflet" | "ns.namedArgLoc"
```

Comment:
Alphabetical?

It would become immediately obvious then that there are 3 categories: nothing, `ns.` and `res.`.